### DISCUSSION TITLE
Having trouble visualizing videos, etc

### DISCUSSION THREAD

**Discussion by RosaliaM21** [bahanonu/calciumImagingAnalysis#111] [Q&A]
Hii, I hope you're well!

I have been trying to get the viewMovieRegistrationTest method with the sample videos we have in CIAtah (I am just starting to set it up!) but it doesn't give me the final capture that I can click-and-drag on; at most, my 'capture' is a dark square. Does this have anything to do with the file format or some new update? My guess is that the file is not being read by the program (?). What do you recommend I do to get it to work?

Thank you in advance!

**Comment by bahanonu** [comment]
@RosaliaM21 Hope you're doing well also! I'll answer this in the issue here: https://github.com/bahanonu/ciatah/issues/114.
